app/entities/Assessment/mock.py

```diff
@@ -17,14 +17,14 @@ def get(
       user = '1',
       rubric = '111',
       timestamp = '111',
-      criteria = [CriterionModel(name='xyz',value='abc')]
+      criteria = [CriterionModel(id='123',value='abc')]
```

Comment:
Please be consistent in formatting. `Constructor(key=val, key=val)` not `Constructor(key = val, key = val)` or `Constructor(key=val,key=val)`

Reply:
I will address this

Reply:
I will address this